build.gradle

```diff
@@ -137,7 +137,7 @@ subprojects {
   }
 }
 
-for ( sv in ['2_9_1', '2_9_2', '2_10_4', '2_11_5'] ) {
```

Comment:
scalatest_2.12 is only released for 3.x branch and there is no scala 2.9.x releases for scalatest 3.x